conn_http.go

```diff
@@ -30,13 +36,17 @@ import (
 )
 
 func dialHttp(ctx context.Context, addr string, num int, opt *Options) (*httpConnect, error) {
-	url := &url.URL{
+	u := &url.URL{
 		Scheme: "http",
 		Host:   addr,
 	}
 
+	query := u.Query()
+	query.Set("default_format", "Native")
+	u.RawQuery = query.Encode()
+
 	if opt.TLS != nil {
-		url.Scheme = "https"
+		u.Scheme = "https"
 	}
 
 	connect := &httpConnect{
```

Comment:
Currently, you use the transport directly vs. a client yet don't exploit this lower level abstraction as far as I can tell?

Reply:
Change to Client with custom Transport